### PR TITLE
[LLM] Extract tests to `.github/actions` to improve reusability

### PR DESCRIPTION
**Pull request by Oscilloscope98**
## Description

LLM actions refactor to extract tests into `.github/actions/llm`

### 1. Why the change?

Extract tests to `.github/actions` to improve reusability for more test cases, e.g. nightly tests, avx2 runner, etc.

### 2. User API changes

N/A

### 3. Summary of the change 

Move tests for convert, inference, langchain, cli, as well as llm test env creation (install test) and llm test enc removing in to  `.github/actions/llm` for better reusability.

### 4. How to test?
- [x] Unit test

### CODE REVIEW

**Review by MeouSker77 — APPROVED** [review]
LGTM

File: .github/actions/llm/cli-test/action.yml [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
   using: "composite"
   steps:
     - name: Test llama llm-cli
       shell: bash
       run: |
-        source $CONDA_HOME/bin/activate bigdl-init-llm
+        source $CONDA_HOME/bin/activate bigdl-llm-test
         $CONDA_HOME/bin/conda info
         
         llm-cli -t $THREAD_NUM -n 256 -x llama -m $LLAMA_INT4_CKPT_PATH -p 'Once upon a time,'
```
Comment (cyita) [inline .github/actions/llm/cli-test/action.yml:13]: Maybe we should check the outputs in the following PR.

Comment (Oscilloscope98) [reply .github/actions/llm/cli-test/action.yml:13]: Yes, we should check the perplexity of test outputs later.

**Review by cyita — APPROVED** [review]
LGTM

**Pull request merged**